lib/common/socketpool.c

```diff
@@ -334,7 +334,8 @@ static void try_connect(h2o_socketpool_connect_request_t *req)
     switch (target->type) {
     case H2O_SOCKETPOOL_TYPE_NAMED:
         /* resolve the name, and connect */
-        req->getaddr_req = h2o_hostinfo_getaddr(req->getaddr_receiver, target->url.host, target->peer.named_serv, AF_UNSPEC,
+        req->getaddr_req = h2o_hostinfo_getaddr(req->getaddr_receiver, target->url.host, target->peer.named_serv, AF_INET,
+        // req->getaddr_req = h2o_hostinfo_getaddr(req->getaddr_receiver, target->url.host, target->peer.named_serv, AF_UNSPEC,
```

Comment:
I believe these changes are unnecessary?

Reply:
Oh I mistakenly committed this change, this is due to some error (I don't remember) when ipv6 address was resolved. Will revert this back.

Reply:
Done in https://github.com/h2o/h2o/pull/3405/commits/5be0daa4f070ad22b7313452ad92809882961257